server/app/src/main/java/com/ams/app/model/Student.java

```diff
@@ -0,0 +1,32 @@
+package com.ams.app.model;
+import java.sql.Timestamp;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.UpdateTimestamp;
+import lombok.Data;
+
+@Table(name="Student")
```

Comment:
Table name should be plural (`students`)

Reply:
changed to students